Formula/sourcery.rb

```diff
@@ -13,11 +13,10 @@ class Sourcery < Formula
   depends_on :xcode => ["8.0", :build]
 
   def install
-    ENV.delete("CC")
-    ENV["SDKROOT"] = MacOS.sdk_path
-    system "swift", "build", "-c", "release"
-    bin.install ".build/release/sourcery"
-    lib.install Dir[".build/release/*.dylib"]
```

Comment:
This is changing the installed files: with your change, we will not be installing the dylibs anymore, but would install the GUI app. Why?